### DISCUSSION TITLE
Кастомные модули

### DISCUSSION THREAD

**Discussion by ArturSharafutdinov** [yandex-maps-unofficial/vue-yandex-maps#357] [Q&A (2.x)] [answered]
Интересует возможность подключения и использования кастомных модулей, например, https://yandex.ru/dev/maps/jsbox/2.1/arrow/

**Comment by daniluk4000** [comment]
Модули в новых Картах есть, и все, что поддерживается в нашей библиотеке, вы можете использовать. Все, чего там нет, не можете :D

**Reply by ArturSharafutdinov** [reply]
У вас нет примера создания и подключения модуля?)

**Reply by daniluk4000** [reply]
А что вы хотите? Мы в Картах под капотом модули от Яндекса подключаем в фоне

Какая задача?

**Reply by ArturSharafutdinov** [reply]
Конкретно сейчас я хочу данный пример (https://yandex.ru/dev/maps/jsbox/2.1/arrow/) повторить с использованием вашей библиотеки

**Reply by daniluk4000** [reply, marked as answer]
Вы можете просто разработать новый компонент и присобачить его на карту. Наша библиотека это позволяет - а для отрисовки использовать YandexMapFeature

Либо при желании использовать API карты, которую вы получаете в v-model, и написать на чистом JS. Так как наша библиотека - обертка, вы можете работать с Яндексом напрямую

Но вспомогательных методов для модулей у vue-yandex-maps нет
